odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/BaseRepositoryTest.java

```diff
@@ -0,0 +1,13 @@
+package org.opendatadiscovery.oddplatform;
+
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+
+/**
+ * Base test class for repository layer
+ *
+ * @author matmalik on 10.12.2021
+ */
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

Comment:
I suppose this annotation is superfluous

Reply:
agree